project/src/main/webapp/create-event.html

```diff
@@ -101,14 +103,14 @@ <h3>Create an Event</h3>
         <div class="col-lg-12">
           <div class="input-group" id="interests-div">
             <div class="input-group-prepend">
-              <label class="input-group-text" for="interests">Labels</label>
+              <label class="input-group-text" for="interests">Interests</label>
             </div>
-            <input
+            <select
```

Comment:
Have you confirmed that input -> select still properly passes stuff into the DB? I remember changing it to input so that the values would go in ok, but I can't quite remember. Can you double check with this code?

Reply:
+1

Reply:
Yup, you're right, done